azure/azuredeploy.json

```diff
@@ -40,7 +40,7 @@
         },
         "numberOfWorkers": {
             "defaultValue": 3,
-            "minValue": 1,
+            "minValue": 0,
```

Comment:
Do we really need to change this? Aren't we already setting it to `0`  for  valgrind  tests in  another place?

Setting it to zero by default could be a bad user experience

Reply:
I initially thought that leaving the `minValue` set to `1` may have problems during some kind of constraint checks, but it did not. Reverted it now

Reply:
Update: if we do not set `minValue` to `0` also, then we get the error below as setting just `numberOfWorkers` to `0` errors out.

`Azure Error: InvalidTemplate
Message: Deployment template validation failed:  'The provided value for the template parameter 'numberOfWorkers' at line '1'`